plugin-locationcomponent/src/test/java/com/mapbox/maps/plugin/locationcomponent/LocationIndicatorLayerWrapperTest.kt

```diff
@@ -162,8 +162,8 @@ class LocationIndicatorLayerWrapperTest {
     layer.updateStyle(newStyle)
     val radius = 1.0
     layer.accuracyRadius(radius)
-    verify(exactly = 0) { style.setStyleLayerProperty(INDICATOR_LAYER_ID, "accuracy-radius", Value.valueOf(radius)) }
-    verify(exactly = 1) { newStyle.setStyleLayerProperty(INDICATOR_LAYER_ID, "accuracy-radius", Value.valueOf(radius)) }
+    verify(exactly = 0) { style.setStyleLayerProperty(INDICATOR_LAYER_ID, "accuracy-radius", any()) }
+    verify(exactly = 1) { newStyle.setStyleLayerProperty(INDICATOR_LAYER_ID, "accuracy-radius", any()) }
```

Comment:
Why isn't the value checked anymore?

Reply:
I guess we can not directly compare double value here, and even though the test passed locally on my device, it consequentially failed on CI.

Reply:
Right, check with [EPS](https://github.com/mapbox/mapbox-maps-android/blob/main/app/src/androidTest/java/com/mapbox/maps/testapp/viewport/ViewportPluginTest.kt#L105) then?

Reply:
Indeed, if this test is checking calling correct style - I'm fine with using any(). Also - other tests here seem to be written similarly, eg `testLayerNotReady`  above checks 1.0 value, rewrite all of them to avoid flucky tests?

Reply:
Discussed offline, we are ok with using `any()` here as test is primarily intended for the callback validation. We need to revisit the `Value` class validation across the codebase, a ticket is created to follow up the tail work.